packages/shared/src/modules/users/libs/types/user-info-request-dto.type.ts

```diff
@@ -0,0 +1,5 @@
+import { type UserInfoResponseDto } from "./user-info-response-dto.type.js";
+
+type UserInfoRequestDto = UserInfoResponseDto;
```

Comment:
I think it's better to duplicate type declaration and avoid this assignment in case types might change in the future (and from semantic perspective too)

Reply:
I assumed that it would contain absolutely all user data (without passwords and id) and would be reused fot it wherever it was needed.

Reply:
For request we only need name, so it should be different from response